arangodb/content.md

```diff
@@ -100,7 +100,7 @@ unix> docker run -e ARANGO_RANDOM_ROOT_PASSWORD=1 arangodb arangod --help
 
 ## Persistent Data
 
-ArangoDB use the volume `/var/lib/arangodb3` as database directory to store the collection data and the volume `/var/lib/arangodb-apps3` as apps directory to store any extensions. These directories are marked as docker volumes.
+ArangoDB use the volume `/var/lib/arangodb3` as database directory to store the collection data and the volume `/var/lib/arangodb3-apps` as apps directory to store any extensions. These directories are marked as docker volumes.
```

Comment:
If this is wrong, the `Dockerfile`'s `VOLUME` definition is as well: https://github.com/arangodb/arangodb-docker/blob/fc8a624249c8916376b863e58ab2584bed3715d5/jessie/3.0.1/Dockerfile#L46
```dockerfile
VOLUME ["/var/lib/arangodb3", "/var/lib/arangodb-apps3"]
```